compiler/src/iree/compiler/ConstEval/JitGlobals.cpp

```diff
@@ -609,8 +611,11 @@ class ProgramBuilder {
   InitializationAnalysis initializationAnalysis;
 };
 
-struct JitGlobalsPass : public JitGlobalsBase<JitGlobalsPass> {
-  JitGlobalsPass(const JitGlobalsOptions &options)
+class JitGlobalsPass final : public impl::JitGlobalsPassBase<JitGlobalsPass> {
```

Comment:
Yeah, this really should be a class but not struct. Thanks for making the change. Can you also move the data members to private? (i.e., add `private:` to l.867)

https://github.com/iree-org/iree/blob/d25712c84d741baf1ba8e4ebeeb6151eb34df4d6/compiler/src/iree/compiler/ConstEval/JitGlobals.cpp#L867-L872

Reply:
Good catch, sure! I moved the data members to private and adjusted the PR description.

I think we could also take a look into moving some of the functions to private but that might be something for a follow up.